ompi/mca/osc/rdma/osc_rdma_active_target.c

```diff
@@ -385,6 +385,8 @@ int ompi_osc_rdma_start_atomic (ompi_group_t *group, int mpi_assert, ompi_win_t
 
     OSC_RDMA_VERBOSE(MCA_BASE_VERBOSE_TRACE, "start group size %d", sync->num_peers);
 
+    sync->type = OMPI_OSC_RDMA_SYNC_TYPE_PSCW;
+
     if (0 == ompi_group_size (group)) {
         /* nothing more to do. this is an empty start epoch */
```

Comment:
I believe you are missing the following initialization, cf. the code in the complete operation.

```suggestion
        /* nothing more to do. this is an empty start epoch */
        sync->peer_list.peers = NULL;
```

Reply:
I agree based on code inspection.

Reply:
@dancejic I squashed this commit down and added you as co-author. Thanks!